pkg/cmd/init/cmd.go

```diff
@@ -78,6 +81,9 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	_ = clusterManagerSet.SetAnnotation("singleton-name", "singletonSet", []string{})
 	o.Helm.AddFlags(singletonSet)
 	cmd.Flags().AddFlagSet(singletonSet)
+	cmd.Flags().StringArrayVar(&o.registrationAuth, "registration-auth", []string{}, "The type of authentication to use for registering and authenticating with hub, this flag can be repeated to specify multiple authentication types.")
+	cmd.Flags().StringVar(&o.hubClusterArn, "hub-cluster-arn", "",
```

Comment:
I think you would need to validate this value

Reply:
Thr validation for this is already added to cluster-manager spec and the validation error is already propagated back to the clusteradm consumer even now. 

Is that sufficient?